JUST.net/Transformer.cs

```diff
@@ -502,7 +502,7 @@ public static bool stringequals(object[] list)
 
             if (list.Length >= 2)
             {
-                if (list[0].ToString().Equals(list[1].ToString()))
+                if (string.Equals(list[0]?.ToString(), list[1]?.ToString(), StringComparison.InvariantCultureIgnoreCase))
```

Comment:
You're assuming 'InvariantCultureIgnoreCase', one might not want this, it's better to make this an option, through EvaluationMode I'd say.

Reply:
@Courela , are you saying that just for the stringequals function or you think it should be the same all across? And please confirm that you mean: if evaluationMode == strict then use CurrentCulture else use InvariantCultureIgnoreCase.

Reply:
It should behave the same across the application.
I meant adding another EvaluationMode, but I was thinking about #131, where EvaluationMode is a Flags' enum. Like it is now, your suggestion seems fine to me.

Reply:
To make this consistent across the application we would have to change the ifcondition comparison to be impacted by the EvaluationMode as well. With that, in the Strict mode, boolean values passed in to the ifcondition call would have to adhere to the boolean string representations, which start with a capital T (True) or F (False). Otherwise the comparison would return false.

For example, in the Strick mode, the following test would fail:

```csharp
var input = "{\"d\": true}";
var transformer = "{ \"result\": \"#ifcondition(#valueof($.d),true,yes,no)\" }";

var context = new JUSTContext  {  EvaluationMode = EvaluationMode.Strict  };
var result = new JsonTransformer(context).Transform(transformer, input);

 Assert.AreEqual("{\"result\":yes}", result);
```

While this one would pass:

```csharp
var input = "{\"d\": true}";
var transformer = "{ \"result\": \"#ifcondition(#valueof($.d),True,yes,no)\" }";

var context = new JUSTContext  {  EvaluationMode = EvaluationMode.Strict  };
var result = new JsonTransformer(context).Transform(transformer, input);

 Assert.AreEqual("{\"result\":yes}", result);
```

Pay close attention to the difference between **true** and **True** in the transformer.

Are you fine with the new behavior in Strick mode?

Reply:
As long as the difference between "Strict" and the default "FallbackToDefault" is well perceived, it's fine by me.
Using .Net has this "feature" for strings made of booleans.